worldedit-bukkit/build.gradle

```diff
@@ -54,27 +54,29 @@ processResources {
 
 jar.archiveName="fawe-bukkit-${project.parent.version}.jar"
 jar.destinationDir = file '../mvn/com/boydti/fawe-bukkit/' + project.parent.version
-task createPom << {
-    pom {
-        project {
-            groupId 'com.boydti'
-            artifactId 'fawe-bukkit'
-            version project.parent.version
+task createPom {
+    doLast {
+        pom {
+            project {
+                groupId 'com.boydti'
+                artifactId 'fawe-bukkit'
+                version project.parent.version
+            }
         }
-    }
```

Comment:
Unnecessary changes

Reply:
Yep the formatting needs to be adjusted. Will do it tomorrow.  I'm going to bed now. Checking that the checkstyle can be used for formatting. Thanks

Reply:
reverted the Gradle changes cause Gradle upgrade is done in commanding branch.